CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 ### Bug Fixes
 
 * Fixed the compile warnings of using deprecated method `RealmProxyMediator.getTableName()` in generated mediator classes (#5455).
+* [ObjectServer] now retrying network query when encountering any `IOException` (#5453).
+* Fixed a `NoClassDefFoundError` due to using `@SafeVarargs` (#5463). 
 
 ### Interal
```

Comment:
Missing `n`

Reply:
are you referring to "a `NoClassDefFoundError`"? I think `a` is more appropriate in this case https://www.englishclub.com/pronunciation/a-an.htm 🤓

Reply:
No I mean `Interal` :D

Reply:
Ah yeah! let's blame @cmelchior

Reply:
Lets not  😛